Testreglar/4.1.2/Nett/nett-4.1.2b.json

```diff
@@ -0,0 +1,314 @@
+			"type": "jaNei",
+			"ruting": {
+				"ja": {
+					"type": "gaaTil",
+					"steg": "3.1"
+				},
+				"nei": {
+					"type": "gaaTil",
+					"steg": "3.11"
+				}
+			}
+		},
+		{
+			"stegnr": "3.1",
+			"spm": "Hvilken knapp tester du?",
+			"ht": "<p>Beskriv knappen, slik at det er mulig å identifisere den i ettertid. Hvis verktøyet finner flere knapper, registrerer du en og en.</p>",
+			"type": "tekst",
+			"ruting": {
+				"alle": {
+					"type": "gaaTil",
+					"steg": "3.2"
+				}
+			},
+			"label": "Knapp:",
+			"multilinje": true
+		},
+		{
+			"stegnr": "3.2",
+			"spm": "Hvilket resultat gir verktøyet for dette skjemaelementet?",
+			"ht": "<p>Verktøyet gir resultat:</p>",
+			"type": "radio",
+			"ruting": {
+				"alt0": {
+					"type": "avslutt",
+					"fasit": "Nei",
+					"utfall": "Knappen har ikke et tilgjengelig navn."
+				},
+				"alt1": {
+					"type": "gaaTil",
+					"steg": "3.3"
+				},
+				"alt2": {
+					"type": "gaaTil",
+					"steg": "3.3"
+				}
+			},
+			"svarArray": [
+				"Brudd (Failed)",
+				"Samsvar (Passed)",
+				"Ingen"
+			]
+		},
+		{
+			"stegnr": "3.3",
+			"spm": "Er knappen kodet med role button?",
+			"ht": "<p>Slik tester du:</p>\n<p><strong>Alternativ 1:</strong></p>\n<ul>\n<li>Åpne DevTools i nettleseren (F12) Chrome</li>\n<li>Inspiser knappen med funksjonen \"inspiser element med mus\"\n<ul>\n<li>Klikk på pil-knappen i øvre venstre hjørne av vinduet, for å aktivere funksjonen</li>\n</ul>\n</li>\n<li>Hold musepekeren over knappen du tester</li>\n<li>En pop-up viser blant annet \"Name\" og \"Role\" under overskriften \"Accessibility\"</li>\n<li>Sjekk om informasjon under \"Role\" er \"button\"</li>\n</ul>\n<p><strong>Alternativ 2:</strong></p>\n<ul>\n<li>Inspiser knappen i Chrome</li>\n<li>Bruk Accessibility Checker verktøy i inspiser</li>\n<li>Under Computed Properties, finner du blant annet \"Name\" og \"Role\"</li>\n<li>Sjekk om informasjon under \"Role\" er \"button\"</li>\n</ul>",
```

Comment:
Hvis du likevel skal inn å inspisere, hvorfor bruker vi QW extension? Det føles ut som om det er lagt til bare for å ha "brukt verktøy" og at det ikke er tenkt på effektivitet. Gi også hjelpetekst til alternativ måte å sjekke det på, som accessibility tab.

Reply:
Steg for bruk av verktøy har blitt fjernet.